digitalstrategie/settings/dev.py

```diff
@@ -32,14 +32,43 @@
 
 if os.getenv("DATABASE") == "postgresql":
     DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'django',
-        'USER': 'django',
-        'PASSWORD': '',
-        'HOST': '',
-        'PORT': '5557',
-        'OPTIONS': {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql',
+            'NAME': 'django',
+            'USER': 'django',
+            'PASSWORD': '',
+            'HOST': '',
+            'PORT': '5557',
+            'OPTIONS': {
             },
         }
     }
+
+# CSP for development (not very strict)
+CSP_DEFAULT_SRC = ["'self'"]
+# unsafe-eval only for testing
+CSP_SCRIPT_SRC = ["'unsafe-eval'"]
+CSP_SCRIPT_SRC_ATTR = ["'none'"]
+# wagtail (and webpack during dev) requires unsafe-inline
+CSP_SCRIPT_SRC_ELEM = ["'self'", "berlin.de",
+                       "www.berlin.de", "'unsafe-inline'"]
+CSP_IMG_SRC = ["'self'", "berlin.de", "www.berlin.de", "www.gravatar.com"]
+CSP_OBJECT_SRC = ["'none'"]
+CSP_MEDIA_SRC = ["'self'"]
```

Comment:
should this not also mention youtube and vimeo or do we not need to given its in iframe?